### PR TITLE
[bug 1126030] Enable CORS for Heartbeat API.

### PR DESCRIPTION
**Pull request by rlr**
r?

### CODE REVIEW

File: fjord/heartbeat/views.py [hunk 1]
```diff
@@ -3,10 +3,18 @@
...
 from fjord.journal.utils import j_error
 
 
 class HeartbeatV2API(rest_framework.views.APIView):
+
+    @classmethod
+    def as_view(cls, **initkwargs):
+        # Enable CORS
+        view = super(HeartbeatV2API, cls).as_view(**initkwargs)
+        return cors_enabled('*', methods=['POST'])(view)
```
Comment (willkg) [inline fjord/heartbeat/views.py]: Nice!

**Comment by gregglind** [comment]
I think POST only is okay here, but fear it's missing something obvious.   We don't secretly allow PUT or want to support API access via GET right?  Dashboards go through some other mechanism?

**Comment by willkg** [comment]
@gregglind: That API endpoint only does POST right now. Unless you need something different, it only needs CORS support for POST.

**Comment by willkg** [comment]
This looks good to me. R+!

**Comment by willkg** [comment]
ba498e3 [bug 1126030] Enable CORS for Heartbeat API.

**Comment by willkg** [comment]
Thank you!

**Pull request merged**